pkg/volume/cinder/attacher.go

```diff
@@ -44,7 +44,8 @@ var _ volume.Attacher = &cinderDiskAttacher{}
 var _ volume.AttachableVolumePlugin = &cinderPlugin{}
 
 const (
-	checkSleepDuration       = 1 * time.Second
+	probeVolumeInitDealy     = 1 * time.Second
```

Comment:
@edisonxiang 
Typo: probeVolumeInitDealy
Should be: probeVolumeInitDelay
Also please rename in other places below.

Reply:
@kabakaev Thanks, I will submit another commit.